### PR TITLE
Elasticity stabilization parameter 

### PR DESCRIPTION
**Pull request by EstherHeck**
This is the new version of PR #3770. 

### Before your first pull request:

* [X] I have read the guidelines in our [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md) document.

### For all pull requests:

* [X] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [X] I have tested my new feature locally to ensure it is correct.
* [X] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/master/tests/) directory.
* [X] I have added a changelog entry in the [doc/modules/changes](../blob/master/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

**Comment by EstherHeck** [comment]
should I rebase with master?
and squash all the commits?

File: tests/elastic_stabilization_time_scale_factor/screen-output [hunk 1]
```diff
@@ -0,0 +1,80 @@
+-----------------------------------------------------------------------------
```
Comment (bobmyhill) [inline tests/elastic_stabilization_time_scale_factor/screen-output:1]: Could you strip out the lines that start with `--`, `+` and `|` from this file?
Alternatively, if you have tested this prm using ctest, you can copy the file
 `aspect/build/tests/output-<test name>/screen-output.notime` into 
 `aspect/build/tests/<test name>/screen-output`.

**Review by bobmyhill — CHANGES_REQUESTED** [review]
Excellent, thanks! Just one change to the test output, than I'll set the tester running. 
If you want to squash some of your commits to make the history a bit tidier, then go ahead, but I wont insist on it.

File: tests/elastic_stabilization_time_scale_factor/screen-output (hunk 1 above)
Comment (EstherHeck) [reply tests/elastic_stabilization_time_scale_factor/screen-output]: done :)

**Review by bobmyhill — APPROVED** [review]

**Review by gassmoeller — APPROVED** [review]
Ok, I think we discussed this issue sufficiently (sorry for missing the past 4 months of the discussion or so :-)), let's go forward with this. If necessary we can adjust the default later (or remove the fixed timestep option at some point? @EstherHeck what is your thought on that?).

Also thanks @EstherHeck  for the incredible amount of testing and patience with this change!

**Comment by EstherHeck** [comment]
Great, that this is finally merged! And yes, I personally don't see a reason for the fixed time step option, but I am also running quite specific models...

**Pull request merged**